src/sim/UI/view.py

```diff
@@ -4,157 +4,74 @@
 from PyQt5.QtGui import * 
 from screeninfo import get_monitors
 
-class MainWindow():
-    def __init__(self, monitorSize, windowTitle="Default"):
-        print("Initializing main window")
+class View():
+    def __init__(self, controller):
         self.__app = QApplication([])
-        self.__window = QWidget() 
-        self.__title = windowTitle
+        self.__window = QWidget()
+        self.__controller = controller
 
-        self.__window.setStyleSheet("background-color: green; border: 5px solid black;")
```

Comment:
Why get rid of background color and style?

Reply:
It overrides button styles and screwed it up. Made it impossible to properly style the button. Also just looks better

Reply:
We can agree to disagree on it makes it look better but no point in focusing on fixing the style if it breaks main functionality

Reply:
yeah, like i said, it broke the ability to make a button that had any size or shape to it